packages/devtools_app/lib/src/standalone_ui/api/vs_code_api.dart

```diff
@@ -78,7 +139,9 @@ abstract interface class VsCodeDevicesEvent {
 abstract interface class VsCodeCapabilities {
   bool get executeCommand;
   bool get selectDevice;
+  bool get openDevToolsFeature;
```

Comment:
can we change this to "openDevToolsPage"? A "feature" makes me think of some item within a page (which has a set of features), but not the page itself.

Reply:
Yeah, I'll change this. I can't recall why I avoided page, but we use page elsewhere to mean the same thing, so we should make this match.

If we add support for jumping to something more specific in future, we'll probably need changes to the API anyway (and probably a new method would be the most compatible way to do that).

Reply:
Done!